.github/workflows/ci-unix-shared-local.yml

```diff
@@ -51,15 +51,6 @@ jobs:
     - run: pip install meson
     - name: Print ImageMagick version
       run: convert --version
-    - name: Set shared libs
-      if: steps.cache-ext.outputs.cache-hit != 'true'
-      working-directory: ./ext
-      run: >
-        sed -i'' -e 's/-DBUILD_SHARED_LIBS=OFF/-DBUILD_SHARED_LIBS=ON/' *.cmd
-
-        sed -i'' -e 's/--default-library=static/--default-library=shared/' dav1d.cmd
-
-        sed -i'' -e 's/ninja yuv/ninja yuv_shared/' libyuv.cmd
```

Comment:
I think we should delete this file, because it is no longer testing locally-built shared libs.

Reply:
Wouldn't we still want to test a shared build of libavif with local dependencies, regardless of how those dependencies are compiled? I think this is the only workflow dedicated specifically to that.

Reply:
Actually yes, you are right: at least, we check that we compile against the recommended versions.

Reply:
You are right. I was confused. Sorry.